### ISSUE TITLE
test代码的算error分母好像不对

### ISSUE THREAD

**Issue by Incandescenced1** [github-luffy/PFLD_68points_Pytorch#44]
分母应该是样本的个数，但是代码写的是len(dataloader）*args. batchsize 这个要比样本数多

**Comment by hs7251912** [comment]
是的  把*args. batchsize删除                                                                                                
loss = loss_sum / len(test_loader)
